scripts/release.js

```diff
@@ -15,14 +29,17 @@ const updatePackageVersion = (nextVersion) => {
 
 const release = async (nextVersion) => {
   try {
+    updateReadme();
     await updateChangelog(nextVersion);
     updatePackageVersion(nextVersion);
     await exec(`git add -u`);
     await exec(
       `git commit -m "docs: release @excalidraw/excalidraw@${nextVersion}  🎉"`,
     );
-    /* eslint-disable no-console */
-    console.log("Done!");
+    // revert readme after release
+    fs.writeFileSync(`${excalidrawDir}/README.md`, originalReadMe, "utf8");
+    // /* eslint-disable no-console */
+    // console.log("Done!");
```

Comment:
revert or remove?

Reply:
revert since I am reverting to original readme